spec/system/user_selects_looking_for_work_spec.rb

```diff
@@ -15,6 +15,7 @@
     perform_enqueued_jobs do
       click_button("SUBMIT")
     end
+    expect(page).to have_text("Your profile was successfully updated")
```

Comment:
One more that just popped up where checking the status of the page before we check the user would be beneficial